src/components/shared/Container.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const Container = ({children}: {children: React.ReactNode}) => {
```

Comment:
To make this component more versatile, you could accept a **className** prop, allowing users to add custom styles in addition to the defaults.